publication/src/test/scala/hmda/publication/reports/disclosure/D51Spec.scala

```diff
@@ -40,11 +40,12 @@ class D51Spec extends WordSpec with MustMatchers with LarGenerators {
   val expectedDispositions = List(ApplicationReceived, LoansOriginated, ApprovedButNotAccepted, ApplicationsDenied, ApplicationsWithdrawn, ClosedForIncompleteness)
 
   "Generate a Disclosure 5-1 report" in {
-    val result = Await.result(D51.generate(source, fips, respId), 5.seconds)
+    val result = Await.result(D51.generate(source, fips, respId, Future("Corvallis Test Bank")), 5.seconds)
```

Comment:
Could this test be done with `AsyncWordSpec` to avoid blocking with `Await.result`?